backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/SetVmTicketCommand.java

```diff
@@ -182,6 +182,12 @@ protected void perform() {
         }
     }
 
+    private void updateConsoleUser(final DbUser user, VmManager vmManager, VmDynamic vm) {
+        vm.setConsoleUserId(user.getId());
+        vm.setConsoleCurrentUserName(getConsoleUserName());
+        vmManager.update(vm);
```

Comment:
In the previous implementation, the script set also `guest_cur_user_name`. Is it OK to leave that out? 
```
SET console_user_id = v_console_user_id,
        guest_cur_user_name = v_guest_cur_user_name,
        console_cur_user_name = v_console_cur_user_name
```

Reply:
yes, it just propagated the guest current user name that was already set in vm-dynamic so it was redundant